simplq/src/components/pages/Admin/ShareQueue.jsx

```diff
@@ -5,7 +5,7 @@ import { ShareButton } from '../../common/Button/Button.stories';
 
 const ShareQueue = (props) => {
   const [clicked, setClicked] = useState(false);
-  const shareUrl = `Hi! Use ${window.location.origin}/j/${props.queueName} to join my queue and get live updates.`;
```

Comment:
Me and @sajmalyousef made this change yesterday. 

Rationale was that people would copy like and paste it on some messenger or something, and this text would be a good idea.

Reply:
Okay, that makes sense.